scylla-cdc-printer/src/printer.rs

```diff
@@ -73,7 +73,9 @@ fn print_row_change_header(data: &CDCRow<'_>) -> String {
     let mut header_to_print = String::new();
     let stream_id = data.stream_id.to_string();
     let (unix_timestamp, _) = data.time.get_timestamp().unwrap().to_unix();
-    let timestamp = NaiveDateTime::from_timestamp(unix_timestamp as i64, 0).to_string();
+    let timestamp = NaiveDateTime::from_timestamp_opt(unix_timestamp as i64, 0)
+        .unwrap()
+        .to_string();
```

Comment:
There is a pre-existing bug here. `to_unix()` returns a tuple of (seconds, nanoseconds), and  `NaiveDateTime::from_timestamp{_opt}` requires both of them - but for some reason the `nanoseconds` from `to_unix` are ignored and `0` is passed instead.

Please send a follow-up fix after this PR is merged.